api/base/serializers.py

```diff
@@ -68,6 +68,12 @@ def to_representation(self, obj):
             ret['self'] = obj.get_absolute_url()
         return ret
 
+
+class LinksField(LinksFieldWIthSelfLink):
```

Comment:
Why is this a subclass of `LinksFieldWithSelfLink`? It uses no behavior from the parent class.